packages/babel-cli/src/babel/dir.js

```diff
@@ -71,7 +71,11 @@ module.exports = function (commander, filenames) {
     _.each(filenames, function (dirname) {
       let watcher = chokidar.watch(dirname, {
         persistent: true,
-        ignoreInitial: true
+        ignoreInitial: true,
+        awaitWriteFinish: {
```

Comment:
Looks like this was added in Chokidar 1.2 and our deps are currently `^1.0.0`, can we bump that up to the current version `^1.6.0`?